src/patterns/data/default/entity_attribute.tsv

```diff
@@ -1,4102 +1,4390 @@
 defined_class	defined_class_name	entity	entity_name	attribute	attribute_name	exact_synonyms	xrefs
-OBA:0000036	body fluid levels	UBERON:0006314	bodily fluid	PATO:0000918	volume
```

Comment:
please make commit in a way that no rows are removed, there should be only additions.